database/index.js

```diff
@@ -89,8 +89,23 @@ const insertIntoMatProd = (dataSet) => {
   return Promise.all(promiseArray);
 }
 
+const getProducts = () => {
+  return new Promise ((resolve, reject) => {
+    return connection.query(`select * from products`, (error, result) => {
+      if(error) {
+        console.log('👎🏽 error getting from products');
+        reject(error)
+      } else {
+        console.log('👍🏽 success in getting data from products');
+        resolve(result);
+      }
+    })
+  })
+}
+
 module.exports.insertIntoProducts = insertIntoProducts;
```

Comment:
@gigicupp If you wanted to, you could shorten this by using a different syntax. You can export an object that contains all of these function inside of it. However, this is not super crucial and if this is  working for you and your workflow, keep going! But maybe if you have time in the future to do a refactor, this would be a quick one.

Reply:
@masoncpott done.`module.exports = {
  insertIntoProducts,
  insertIntoLocations,
  insertIntoMaterials,
  insertIntoLocProd,
  insertIntoMatProd,
  getProducts
};`